astroquery/linelists/cdms/tests/test_cdms_remote.py

```diff
@@ -49,7 +50,25 @@ def test_remote_regex():
 
     assert isinstance(tbl, Table)
     assert len(tbl) == 557
-    assert set(tbl.keys()) == set(['FREQ', 'ERR', 'LGINT', 'DR', 'ELO', 'GUP',
-                                   'TAG', 'QNFMT', 'Ju', 'Jl', "vu", "vl", "Ku", "Kl", "F", "name"])
+    assert set(tbl.keys()) == colname_set
 
     assert set(tbl['name']) == {'H2CN', 'HC-13-N, v=0'}
+
+
+@pytest.mark.remote_data
+def test_2375():
```

Comment:
use names (and PR titles) that are descriptive. No one will remember what issue 2375 was when this test starts failing :)

Reply:
test_complicated_molecule_parsing, then?  (but link #2375 in the docstring?)

Reply:
yeap, I went with `test_molecule_with_parens`, and agree, keeping the reference to the issue number is super useful